man/pam_systemd.xml

```diff
@@ -262,6 +262,51 @@
     based on the <varname>$DISPLAY</varname> variable.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Session limits</title>
+
+    <para>PAM modules earlier in the stack, that is those that come before pam_systemd.so, can set session scope limits using the PAM context objects.
```

Comment:
`pam_systemd.so` should probably be written as `<command>pam_systemd.so</command>`